src/svg-renderer.js

```diff
@@ -366,32 +398,39 @@ class SvgRenderer {
     }
 
     /**
-     * Draw the SVG to a canvas.
+     * Synchronously draw the loaded SVG to this renderer's `canvas`.
+     * @param {number} [scale] - Optionally, also scale the image by this factor.
+     */
+    draw (scale) {
+        if (this._cachedImage === null) throw new Error('SVG image has not finished loading');
```

Comment:
Did you mean to replace checks for _cachedImage === null with this.loaded = true?

Reply:
Good catch; fixed!